api/envoy/api/v2/cluster/outlier_detection.proto

```diff
@@ -24,9 +24,9 @@ option (udpa.annotations.file_status).package_version_status = FROZEN;
 // more information on outlier detection.
 // [#next-free-field: 21]
 message OutlierDetection {
-  // The number of consecutive 5xx responses or local origin errors that are mapped
-  // to 5xx error codes before a consecutive 5xx ejection
-  // occurs. Defaults to 5.
+  // The number of consecutive 5xx responses or local origin errors (e.g. for TCP traffic,
```

Comment:
I think the doc callouts in RST are fine, but the API comment changes are misleading, as they over-index on the TCP case.

Reply:
That makes sense to me. But from my perspective, part of the issue in the first place was that this was named `5xx` which I immediately assumed meant HTTP traffic only. Is there other wording that we can compromise on?

Maybe:
```
The number of consecutive server-side error responses (for HTTP traffic,
5xx responses; for TCP traffic, connection failures; for Redis, failure to respond
PONG, etc.) before a consecutive 5xx ejection occurs. Defaults to 5.
```

I think also L3 traffic is supported but I don't have any experience with that to know the correct wording to use for when that fails 😓

Reply:
Let's go with that.